roles/bootstrap/tasks/root_tasks.yml

```diff
@@ -18,6 +18,9 @@
     - { name: 'vm.swappiness', value: 0 }
     - { name: 'net.ipv4.tcp_syncookies', value: 0 }
     - { name: 'fs.file-max', value: 1000000 }
+    - { name: 'net.ipv4.tcp_tw_reuse', value: 1 }
+    - { name: 'net.ipv4.tcp_tw_recycle', value: 1 }
```

Comment:
tcp_tw_recycle is not recommended, https://github.com/pingcap/tidb-ansible/blob/master/roles/bootstrap/tasks/root_tasks.yml#L11

Reply:
L11 mean remove `tcp_tw_recycle` when setting 0(disable).
It's a good option that enables it for fast recycling of TIME_WAIT sockets.

Reply:
See https://mp.weixin.qq.com/s/ydPhvFEhrol-8qYxp4LFog